### PR TITLE
API Endpoints Working

### PR DESCRIPTION
**Pull request by mayankkumar2**
I've added all the required features in the flask app. 
I will soon make docs for the API endpoints. 
I have deployed the App on Heroku at http://vitevents.herokuapp.com/

### CODE REVIEW

**Review by Geek-ubaid — CHANGES_REQUESTED** [review]
Suggested changes to be done:
- [ ] Introduce modularity in the code 
- [ ] Use proper variable name for better readability

File: Aptfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+libsm6
+libxrender1
+libfontconfig1
+libice6
```
Comment (Geek-ubaid) [inline Aptfile:4]: Why this file is pushed?

File: README.md [hunk 2]
```diff
@@ -1,59 +1,111 @@
...
+git init
+heroku create
+git add .
+git commit -m "<YOUR MESSAGE>"
+git push heroku master
 ```
+* API endpoints
+		1.  /crop
+		2. /cropb64
+		3. /getTable
```
Comment (Geek-ubaid) [inline README.md:86]: NO need to write it here. Prepare the docs and add link in the readme in the badge as given in the template.

**Review by Geek-ubaid — CHANGES_REQUESTED** [review]

**Comment by mayankkumar2** [comment]
@Geek-ubaid
 I've increased the modularity of the code, I've changed some variable names but most the variable names appeared self-explanatory to me.

File: README.md (hunk 2 above)
Comment (mayankkumar2) [reply README.md:86]: I'll change it once the docs are done.

File: Aptfile (hunk 1 above)
Comment (mayankkumar2) [reply Aptfile:4]: These are the configurations required to deploy the app to Heroku.

**Review by Geek-ubaid — APPROVED** [review]

**Comment by Geek-ubaid** [comment]
Suggested changes to be done:
- [X] Introduce modularity in the code 
- [X] Use proper variable name for better readability

**Pull request merged**